lib/linguist/languages.yml

```diff
@@ -4227,6 +4227,11 @@ Smarty:
   codemirror_mime_type: text/x-smarty
   tm_scope: text.html.smarty
   language_id: 353
+Solidity:
+  type: programming
+  ace_mode: text
+  tm_scope: source.solidity
+  language_id: 237469032
```

Comment:
Could you add a color for Solidity? We usually try to use colors from official logos if any.